### PR TITLE
RDM-4473: Add AppInsights to ccd-case-management-web

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### JIRA link  ###

https://tools.hmcts.net/jira/browse/RDM-4473

### Change description ###

Added AppInsights to ccd-case-management-web

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,9 +62,13 @@
     "@hmcts/ccpay-web-component": "4.0.0-beta87",
     "@hmcts/media-viewer": "1.3.1",
     "@hmcts/nodejs-healthcheck": "1.7.0",
+    "@hmcts/nodejs-logging": "^3.0.0",
+    "@hmcts/properties-volume": "0.0.9",
     "@nguniversal/express-engine": "^6.0.0",
     "@nguniversal/module-map-ngfactory-loader": "^6.0.0",
     "@nicky-lenaers/ngx-scroll-to": "^1.0.0",
+    "applicationinsights": "^1.0.8",
```
Comment (MSancaktutar) [inline package.json:70]: Could this be better a higher version?

**Review by MSancaktutar — APPROVED** [review]
Minor comment, otherwise good. Thanks!

**Review by MSancaktutar — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (rajaram-nagarajan) [reply package.json:70]: This upgrade will impact many other dependencies. So, we can consider upgrading this along with ccd-admin-web or ccd-gateway.

**Pull request merged**